openedx/core/djangoapps/user_authn/templates/user_authn/edx_ace/passwordreset/email/body.html

```diff
@@ -37,22 +37,7 @@ <h1>
                 {% trans "Change my Password" as course_cta_text %}
 
                 {# email client support for style sheets is pretty spotty, so we have to inline all of these styles #}
-                <a href="{{ reset_link }}" style="
-                    color: #ffffff;
-                    text-decoration: none;
-                    border-radius: 4px;
-                    -webkit-border-radius: 4px;
-                    -moz-border-radius: 4px;
-                    background-color: #005686;
-                    border-top: 12px solid #005686;
-                    border-bottom: 12px solid #005686;
-                    border-right: 50px solid #005686;
-                    border-left: 50px solid #005686;
-                    display: inline-block;
-                ">
-                    {# old email clients require the use of the font tag :( #}
-                    <font color="#ffffff"><b>{{ course_cta_text }}</b></font>
-                </a>
+                {% include "ace_common/edx_ace/common/return_to_course_cta.html" with course_cta_text=course_cta_text reset_url=reset_link %}
```

Comment:
shouldn't this be `course_cta_url=reset_link` instead of `reset_url=reset_link`

Reply:
Well in password reset email.. we have reset link having token generated ie **/password_reset_confirm/hash/?track=pwreset**. I dont think we have any other thing inside url.

Reply:
My bad, I looked at the `return_to_course_cta` button and this makes sense